test/fixtures/es-modules/package-type-module/wasm-dep.mjs

```diff
@@ -0,0 +1,13 @@
+import { strictEqual } from 'assert';
+
+export function jsFn () {
+  state = 'WASM JS Function Executed';
+  return 42;
+}
+
+export let state = 'JS Function Executed';
+
+export function jsInitFn () {
+  strictEqual(state, 'JS Function Executed');
+  state = 'WASM Start Executed';
+}
```

Comment:
nit: It'd be nice to have some output to validate that the code is actually run.
```suggestion
}

console.log('Hello from WASM');

```

Reply:
Does this suggested `console.log` in a JS file validate that the Wasm has run?

Reply:
Yes, because it tells us the WASM has imported it. Another option is to move it to one of the exported functions.